clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -2412,15 +2410,15 @@ public void testReturnRecordsDuringRebalance(GroupProtocol groupProtocol) throws
         ConsumerMetadata metadata = createMetadata(subscription);
```

Comment:
I looped `testReturnRecordsDuringRebalance` and see the following error:

```
Gradle Test Run :clients:test > Gradle Test Executor 253 > KafkaConsumerTest > testReturnRecordsDuringRebalance(GroupProtocol) > "testReturnRecordsDuringRebalance(GroupProtocol).groupProtocol=CLASSIC" FAILED
    org.opentest4j.AssertionFailedError: expected: <11> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:531)
        at app//org.apache.kafka.clients.consumer.KafkaConsumerTest.testReturnRecordsDuringRebalance(KafkaConsumerTest.java:2440)
```

@frankvicky could you please check it?